openedx.yaml

```diff
@@ -14,4 +14,16 @@ oeps:
     oep-5:
         state: False
         reason: TODO - Implement for this application if appropriate
+    oep-7:
+        state: False
+        reason: TODO - Implement for this application if appropriate
+    oep-18:
+        state: False
```

Comment:
This OEP (Python dep. mgmt.) should actually be `true` for this repo (you can tell from the `.in` files in the `requirements/` directory).